Cargo.toml

```diff
@@ -55,6 +55,8 @@ rounding = ["lexical-core/rounding"]
 ryu = ["lexical-core/ryu"]
 # Use the `std` library.
 std = ["lexical-core/std"]
+# Depend on an allocator when building without feature std
+alloc = []
```

Comment:
The high-level crate is not meant to be used in a `no_std` environment. You can use `lexical-core` for that. The API as of v0.6 is also nearly identical to the high-level crate, and only depends on an allocator when `radix` and `correct` are both enabled (see the algorithm section of lexical-core's README for an extended explanation of why, it has to do with internal big integer arithmetic).